### ISSUE TITLE
Python 3.7 DeprecationWarning collections.abc

### ISSUE THREAD

**Issue by tonal** [vxgmichel/aiostream#28]
```
/usr/local/lib/python3.7/site-packages/aiostream/aiter_utils.py:5: DeprecationWarning: Using or importing the ABCs from 'collections' instead of from 'collections.abc' is deprecated, and in 3.8 it will stop working
  from collections import AsyncIterator

/usr/local/lib/python3.7/site-packages/aiostream/core.py:5: DeprecationWarning: Using or importing the ABCs from 'collections' instead of from 'collections.abc' is deprecated, and in 3.8 it will stop working
  from collections import AsyncIterable, Awaitable
/usr/local/lib/python3.7/site-packages/aiostream/core.py:5: DeprecationWarning: Using or importing the ABCs from 'collections' instead of from 'collections.abc' is deprecated, and in 3.8 it will stop working
  from collections import AsyncIterable, Awaitable
```

**Comment by tonal** [comment]
see https://github.com/scikit-learn/scikit-learn/issues/11121 and https://github.com/scikit-learn/scikit-learn/pull/11431

**Comment by tonal** [comment]
Or https://github.com/numpy/numpy/pull/10743

**Comment by vxgmichel** [comment]
Thanks for the report!

It's been fixed in PR #27 and will become visible with version 0.3.2.
